src/v2/Apps/Order/Routes/NewPayment/index.tsx

```diff
@@ -311,7 +296,7 @@ export class NewPaymentRoute extends Component<
 }
 
 export const NewPaymentFragmentContainer = createFragmentContainer(
-  injectCommitMutation(injectDialog(NewPaymentRoute)),
+  createStripeWrapper(injectCommitMutation(injectDialog(NewPaymentRoute)) as any),
```

Comment:
Would `as NewPaymentProps` be sufficient?

Reply:
Nope